build/cmake/install.cmake

```diff
@@ -36,6 +36,8 @@ if(MSVC)
     )
 endif()
 
+wx_get_install_dir(library "lib")
```

Comment:
I think you can also use `${library_dir}` in the `WIN32_MSVC_NAMING` section instead of the current `lib/${wxPLATFORM_LIB_DIR}`.

And `${wxSETUP_HEADER_PATH}` is still installed in `"lib/wx/include"`, should this also be `${library_dir}`? What does configure do?